test/utils/test-utils.js

```diff
@@ -29,18 +29,15 @@ const hyphenToUpperCase = (name) => {
  *
  * @param {String} testCase The path to folder that contains the webpack.config.js
  * @param {Array} args Array of arguments to pass to webpack
- * @param {Boolean} setOutput Boolean that decides if a default output path will be set or not
  * @param {Array<string>} nodeOptions Boolean that decides if a default output path will be set or not
  * @param {Record<string, any>} env Boolean that decides if a default output path will be set or not
  * @returns {Object} The webpack output or Promise when nodeOptions are present
  */
-const run = (testCase, args = [], setOutput = true, nodeOptions = [], env) => {
+const run = (testCase, args = [], nodeOptions = [], env) => {
```

Comment:
`nodeOptions` and `env` should be merged into `options` and let's use execa, no need `spawnSync`

Reply:
yep no need

Reply:
wip